_data/method_content/cognitive-walkthrough.md

```diff
@@ -0,0 +1,19 @@
+## How to do it
```

Comment:
[thought] Hmm, I see what you mean about this feeling weird to be in `_data` but not appropriate to `/content` either.  How do you feel about `_includes/content/` ?  There has been precedence set there for other guides already.  I can see it working in data, too, I just typically don't think of "data" as markdown files and personally wouldn't look there straightaway.

Reply:
good idea!